libr/util/list.c

```diff
@@ -546,10 +546,81 @@ R_API void r_list_insertion_sort(RList *list, RListComparator cmp) {
 	}
 }
 
+void r_list_iterative_merge_sort(RList *list, RListComparator compare) {
```

Comment:
you can drop this and test the master solution? it fixed the stack exhaustion issues.